modules/ROOT/pages/ai.adoc

```diff
@@ -0,0 +1,102 @@
+= AI Assistant plugin
+:navtitle: AI
+:description: Sends queries to registered AI APIs and sets the results in the document or temporarily stores the results for later use.
+:description_short: Query AI APIs and accept results.
+:keywords: plugin, ai, artificial intelligence, query, search.
+:pluginname: AI Assistant
+:plugincode: ai
+:plugincategory: premium
+
+include::partial$misc/admon-ai-pricing.adoc[]
+
+include::partial$misc/admon-requires-6.6v.adoc[]
+
+The {pluginname} allows a {productname} document to send queries to registered AI APIs and either:
+
+. add the returned results to a document; or
+. temporarily store the returned results for later use.
+
+[IMPORTANT]
+.On the absence of an AI Assistant demo
+====
+This initial release of the AI Assistant developer documentation does not include an in-page working demo.
+
+The logistics of setting up such a demo are still being investigated.
+
+When a solution that is both useful and workable is found, it will replace this notice.
+====
+
+== Basic setup
+
+To add the {pluginname} plugin to the editor, add both `{plugincode}` to the `plugins` option in the editor configuration and the `ai_request` function to the editor configuration.
+
+For example:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'ai',
+  toolbar: 'aidialog aishortcuts',
+  ai_request: (request) => new Promise()
+});
+----
+
+[IMPORTANT]
+.Default strings added to all prompts
+====
+The {pluginname} automatically appends default strings to all prompts.
+
+Depending on the prompt, the added string will be one of the following:
+
+[source,text]
+----
+Answer the question based on the context below.
+The response should be in HTML format.
+The response should preserve any HTML formatting, links, and styles in the context.
+----
+
+These added strings both improve the UX and increase the response accuracy.
+
+However, these added strings do increase token use.
```

Comment:
The HTML format also increases the token use

Reply:
To quote my [commit message](https://github.com/tinymce/tinymce-docs/commit/130d03ba3d164763fa850050240b30049d5f0393):

> Added note regarding html format increasing token use.